app/services/user_alerts/alert_user_about_new_device.rb

```diff
@@ -62,7 +62,7 @@ def self.sign_in_events_start_time(user:)
       [
         user.sign_in_new_device_at,
         (IdentityConfig.store.new_device_alert_delay_in_minutes * 3).minutes.ago,
-      ].max
+      ].min
```

Comment:
Opens event window for alert_delay * 3 minutes or new_device_at: whichever is the lesser time.